app/views/notification_mailer/new_editor_email.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= t('mailer.new_editor', title: @lecture.title_with_teacher, username: @username) %>
+<br>
+<!--- <%= t('mailer.subscribe_editor') %> --->
+<p class="pt-2">
+  <%= link_to(t('notifications.edit_lecture'),
```

Comment:
The entry in the locale file for `notifications.edit_lecture` seems to be missing.